natlas-server/app/admin/routes.py

```diff
@@ -19,11 +21,12 @@ def admin():
 		for fieldname, fieldvalue in configForm.data.items():
 			if fieldname.upper() in ["SUBMIT", "CSRF_TOKEN"]:
 				continue
-			if fieldname.upper() == "ELASTICSEARCH_URL" and fieldvalue != current_app.config["ELASTICSEARCH_URL"]: # if we've got a new elasticsearch address, update our current handle to elastic
+			# if we've got a new elasticsearch address, update our current handle to elastic
+			if fieldname.upper() == "ELASTICSEARCH_URL" and fieldvalue != current_app.config["ELASTICSEARCH_URL"]:
 				current_app.elastic = Elastic(fieldvalue)
 			current_app.config[fieldname.upper()] = fieldvalue
 			confitem = ConfigItem.query.filter_by(name=fieldname.upper()).first()
-			confitem.value=str(fieldvalue)
+			confitem.value = str(fieldvalue)
 			db.session.add(confitem)
```

Comment:
Does flask have built in sanity checking on this stuff?

Reply:
Every time I see db.session.add(blah). I starting seeing weird ways to sqli.

Reply:
Yes, we never write SQL and sqlalchemy has tons of sanitation built in. If natlas has a sqli then so does every other app that uses sqlalchemy. At which point we just update sqlalchemy.